src/javascript/jest.ts

```diff
@@ -766,15 +766,15 @@ export class Jest extends Component {
   }
 
   /**
-   * Adds a a setup file to setupFiles configuration array.
+   * Adds a a setup file to Jest's setupFiles configuration, written to package.json.
```

Comment:
Just that. The user can choose where the config goes.
```suggestion
   * Adds a a setup file to Jest's setupFiles configuration
```